src/main/java/it/gov/pagopa/pu/bff/controller/OrganizationController.java

```diff
@@ -1,28 +1,31 @@
 package it.gov.pagopa.pu.bff.controller;
 
-import it.gov.pagopa.pu.bff.controller.generated.BrokersApi;
-import it.gov.pagopa.pu.bff.dto.generated.ConfigFE;
+import it.gov.pagopa.pu.bff.controller.generated.OrganizationsApi;
+import it.gov.pagopa.pu.bff.dto.generated.OrganizationDTO;
 import it.gov.pagopa.pu.bff.security.SecurityUtils;
-import it.gov.pagopa.pu.bff.service.organization.BrokerServiceImpl;
+import it.gov.pagopa.pu.bff.service.organization.OrganizationServiceImpl;
 import lombok.extern.log4j.Log4j2;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.RestController;
 
-@Log4j2
+import java.util.List;
+import java.util.Objects;
+
 @RestController
-public class OrganizationController implements BrokersApi {
+@Log4j2
+public class OrganizationController implements OrganizationsApi {
 
-  private BrokerServiceImpl service;
+  private final OrganizationServiceImpl organizationService;
 
-  public OrganizationController (BrokerServiceImpl service){
-    this.service = service;
+  public OrganizationController(OrganizationServiceImpl organizationService) {
+    this.organizationService = organizationService;
   }
 
   @Override
-  public ResponseEntity<ConfigFE> getBrokerConfig () {
-    log.info("User requested getBrokerConfig()");
-    return new ResponseEntity<>(service.getBrokerConfig(SecurityUtils.getLoggedUser(),SecurityUtils.getAccessToken()), HttpStatus.OK);
+  public ResponseEntity<List<OrganizationDTO>> getOrganizations() {
+    log.info("User requested getOrganizations()");
+    return new ResponseEntity<>(organizationService.getOrganizations(Objects.requireNonNull(SecurityUtils.getLoggedUser()), SecurityUtils.getAccessToken()), HttpStatus.OK);
```

Comment:
This is not required! Spring Security configuration will handle it.
Otherwise you should handle the exception thrown if empty (this way you was returning a 500, when the error was a client mistake, but SpringSecurity will handle it returning 403)
```suggestion
    return new ResponseEntity<>(organizationService.getOrganizations(SecurityUtils.getLoggedUser(), SecurityUtils.getAccessToken()), HttpStatus.OK);
```